src/SDK/SmartDetectors.Extensions/Presentation/AlertExtensions.cs

```diff
@@ -386,7 +386,7 @@ private static string PropertyValueToString(object propertyOwner, PropertyInfo p
                 }
 
                 string linkText = uriFormatterAttribute.LinkText.EvaluateInterpolatedString(propertyOwner);
-                return $"<a href=\"{uriValue}\">{linkText}</a>";
```

Comment:
please publish new nuget version

Reply:
and consume it in the central runtime env

---
In reply to: [233428939](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/24#discussion_r233428939) [](ancestors = 233428939)